subsystems/sonarSensor.py

```diff
@@ -1,9 +1,6 @@
 import wpilib
 import json
-try:
```

Comment:
We definitely need a plan of action for this.   I don't think we have the udp_channel library pip installed on the robot.    It is possible, but we'll need know what version and start testing with it regularly.